src/presentation/controllers/v1/state-machine-controller.ts

```
@@ -0,0 +1,302 @@
+  app.get(
+    '/machines',
+    {
+      schema: {
+        description: 'List state machines',
+        tags: ['State Machines'],
+        response: {
+          200: ListStateMachinesResponseBodySchema,
+        },
+      },
+    },
+    async (request, reply) => {
```

ESLint

[error] 'reply' is defined but never used.

(@typescript-eslint/no-unused-vars)